docs/manager/README.md

```diff
@@ -562,6 +562,7 @@ Check [CLI documentation](/manager/#x-ray-autotracing-cli) for a description of
 #### Requirements
 
 * CloudLinux OS Shared Pro or CloudLinux OS Solo
+* cPanel or Plesk Control Panel
```

Comment:
This part of the docs relates to X-Ray Autotracing feature, not AccelerateWP.
X-Ray and Autotracing are supported for all panels, including panel integration case.
Please, remove this line.

Reply:
done